libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -73,6 +79,16 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("VOLT2_MIN",     5,     AP_Arming,  _min_voltage[1],  0),
 
+    // @Param: RUDDER
+    // @DisplayName: Rudder Arming
+    // @Description: Allow arm/disarm by rudder input. When enabled arming can be done with right rudder, disarming with left rudder. Rudder arming only works in manual throttle modes with throttle at zero +- deadzone (RCx_DZ)
+    // @Values: 0:Disabled,1:ArmingOnly,2:ArmOrDisarm
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS_FRAME("RUDDER",        6,     AP_Arming, _rudder_arming, ARMING_RUDDER_DEFAULT, AP_PARAM_NO_SHIFT, AP_PARAM_FRAME_PLANE |
```

Comment:
I'm pretty sure AP_PARAM_NO_SHIFT here is wrong, from the comments and a vague recollection it's only used for index 0 in the table, which this is not.